protoc-gen-twirp/generator.go

```diff
@@ -397,13 +397,11 @@ func (t *twirp) generateUtils() {
 	t.P(`}`)
 	t.P()
 
-	t.P(`// closebody closes a response or request body and just logs `)
-	t.P(`// any error encountered while closing, since errors are `)
-	t.P(`// considered very unusual.`)
+	t.P(`// closebody closes a response body and ignores any error encountered`)
+	t.P(`// encountered while closing, intended for use in defers to avoid leaks.`)
 	t.P(`func closebody(body io.Closer) {`)
-	t.P(`  if err := body.Close(); err != nil {`)
-	t.P(`    `, t.pkgs["log"], `.Printf("error closing body: %q", err)`)
-	t.P(`  }`)
+	t.P(`  err := body.Close()`)
```

Comment:
How about `_ = body.Close()`, possibly with the comment from the next line?

Reply:
Or, sorry, maybe it's done this way to get around `errcheck -blank`? I've used `_ = body.Close()` or similar before to appease errcheck but maybe `-blank` is a mode I'm not familiar with.

Reply:
Correct, this is done to dodge `errcheck -blank`, which is "errcheck, and I don't want any funny business like assigning to `_`".

Which is exactly what we're doing, of course. But I think this is simpler than telling an opinionated user to change their linter setup to not invoke `errcheck -blank`.